lms/templates/admin/application_instance/show.html.jinja2

```diff
@@ -21,6 +21,15 @@
         </label>
     {% endcall %}
 {% endmacro %}
+{% macro setting_checkbox(label, setting, field_name) %}
```

Comment:
Adding a new macro to handle the settings using JSonSetting to make it easier to do one by one.